### DISCUSSION TITLE
Crypto C++ 版本不一致导致模型初始化det 卡住

### DISCUSSION THREAD

**Discussion by XiaoDongGuoGuo** [PaddlePaddle/PaddleOCR#13789] [Q&A]
![image](https://github.com/user-attachments/assets/f69109c7-3015-41df-9ec8-516653775fe4)
![image](https://github.com/user-attachments/assets/efde91cf-85b4-406d-a36d-5af2660ef18f)
此预测库中加密包Crypto 版本是多少，在哪里可以看到？

**Comment by GreatV** [comment]
https://github.com/PaddlePaddle/Paddle 主框架的库中找找

**Reply by XiaoDongGuoGuo** [reply]
感谢，已找到版本
